apps/emqx_mongodb/src/emqx_mongodb.erl

```diff
@@ -140,7 +140,7 @@ mongo_fields() ->
         {srv_record, fun srv_record/1},
         {pool_size, fun emqx_connector_schema_lib:pool_size/1},
         {username, fun emqx_connector_schema_lib:username/1},
-        {password, fun emqx_connector_schema_lib:password/1},
+        {password, emqx_connector_schema_lib:password_field()},
```

Comment:
Nit: may be a bit confusing lack of uniformness of API, `mod:fun1()` vs `fun mod:fun2/1`

Reply:
Agree here, that's kinda unfortunate. I added a `field` suffix which follows a module-level convention of naming field schemas in literal `map()` form, but it's indeed suboptimal. I honestly currently have no idea what to do with this without making a lot of changes across a lot of apps, once more.